src/main/java/com/google/sps/data/backgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
+/** Interface for the backgroundTaskManager class. */
+public interface backgroundTaskManagerInterface { 
+/**
+   * Checks VM status to make sure they are healthy. 
+   */
+    void checkVM();
+/**
+   * Removes attendee if they haven't been polled in 2 minutes.
+   * Nothing associated with if the attendee is the controller.  
```

Comment:
where is the method that deals with that then?

Reply:
Also mentioned by Chuck, he asked me to change some details to this as well which you'll see in the next PR